app/src/main/java/com/github/sdpteam15/polyevents/user/UserInterface.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.sdpteam15.polyevents.user
+
+import com.github.sdpteam15.polyevents.database.FirebaseUserInterface
+
+/**
+ *  Application user interface
+ */
+interface UserInterface {
+    /**
+     * Associates FirebaseUser
+     */
+    val FirebaseUser : FirebaseUserInterface
+
+    /***
+     * Application user profile List
+     */
+    val ProfileList : List<ProfileInterface>
+
+    /**
+     * Application user name
+     */
+    val Name : String
```

Comment:
If a single user can have multiple profiles on the app, can this user also have multiple usernames ? And so this Name property should be removed to avoid redundancy with the Name from ProfileInterface ?